src/include/network/lan/wireless.rb

```diff
@@ -1123,27 +1123,31 @@ def WirelessKeysDialog
 
     # -------------------- WPA EAP --------------------
 
+    # `RadioButtonGroup uses CurrentButton instead of Value, grrr
```

Comment:
:)

Reply:
Actually not my comment. Just Copy and paste :) object CWM has it already handled - https://github.com/yast/yast-yast2/blob/master/library/cwm/src/lib/cwm/common_widgets.rb#L290